src/Components/ProducerList.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import ProducerListItem, { ProducerListItemProps } from './ProducerListItem';
+import styled from 'styled-components';
+import Table from 'react-bootstrap/Table';
+
+const ListContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    margin: auto;
+`;
+
+class ProducerList extends React.Component {
+    render() {
+        const mockedListData: ProducerListItemProps[] = [
+            {
+                imageSrc: "https://picsum.photos/250",
+                name: "Johnny's Jams",
+                description: "Johnny's Jams: We got the jams. For real.",
+                numProducts: 8,
+            },
+            {
+                imageSrc: "https://picsum.photos/250",
+                name: "Mary Jane's Cabbage",
+                description: "Mary Jane's specializes in potent and pure cabbage for the everyday cabbage consumer.",
+                numProducts: 12,
+            }
+        ]
+
+        return (
+            <ListContainer>
+                <h2>Producer List</h2>
+                <Table striped hover variant="dark">
```

Comment:
I actually think we should avoid using `<table></table>` elements for our list components. 
I read some things about it and found out that using `div` and `ul` unordered lists is better. 
https://www.smashingmagazine.com/2009/04/from-table-hell-to-div-hell/

You can check out the article. But a tldr is basically:
- different browsers and devices render things differently, especially tables, but div are simpler components
- div lists are more readable
```
<table cellpadding="0" cellspacing="0" border="0">
    <tr>
        <td colspan="3" height="120px">....</td>
    </tr>
    <tr>
        <td class="menu" valign="top">...</td>
        <td class="content" valign="top">...</td>
        <td class="aSide" valign="top">...</td>
    </tr>
    <tr>
        <td colspan="3">...</td>
    </tr>
</table>
```
vs
```
<div id="header">...</div>
<div id="menu">...</div>
<div id="content">...</div>
<div id="aSide">...</div>
<div id="footer">...</div>
```
```

Reply:
and since we are using React and styled components, we can make some reusable div lists